web-admin/src/features/navigation/TopNavigationBar.svelte

```diff
@@ -144,12 +145,19 @@
   $: visualizationPaths = visualizations.reduce((map, resource) => {
     const name = resource.meta.name.name;
     const isMetricsExplorer = !!resource?.explore;
+    const groupId = isMetricsExplorer ? "explore" : "canvas";
     return map.set(name.toLowerCase(), {
       label:
         (isMetricsExplorer
           ? resource?.explore?.spec?.displayName
           : resource?.canvas?.spec?.displayName) || name,
-      section: isMetricsExplorer ? "explore" : "canvas",
+      section: groupId,
+      resourceKind: isMetricsExplorer
+        ? ResourceKind.Explore
+        : ResourceKind.Canvas,
+      groupId,
+      groupLabel: isMetricsExplorer ? "Explore" : "Canvas",
+      groupOrder: isMetricsExplorer ? 1 : 0,
```

Comment:
Not sure why we had a map for this in the 1st place, but an array of array feels like a better fit than a map with groupId. Check `SearchableMenuContent.svelte` for reference.